feat: add default support for sphinx-llm (#590)
canonical/sphinx-docs-starter-pack: docs/reference/rst-syntax.rst

.. meta::

>     :description: Find examples of the reStructuredText syntax used in Canonical documentation files.

Reviewer: ```suggestion
    :description: Reference for the reStructuredText syntax conventions used by Canonical.
```

Author: https://github.com/canonical/sphinx-docs-starter-pack/pull/590/commits/47f855515d8142df3c7341d701d0d9e8c544aecc
